kinder asp.net/Models/Item.cs

```diff
@@ -12,16 +12,6 @@ namespace kinder_app.Models
 {
     public class Item : IComparable<Item>, IEquatable<Item>
     {
-        public Item()
```

Comment:
these lines shouldn't be the ones responsible for conflict, try renaming its directory